### PR TITLE
Add SeleniumTest project

### PR DESCRIPTION
**Pull request by Knarik-Dabaghyan**

### CODE REVIEW

File: seleniumTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        this.driver = driver;
+        driver.manage().window().maximize();
+        driver.get("https://google.com");
+        Thread.sleep(3000);
+        WebElement element = driver.findElement(By.name("q"));
+        String expectedName = "selenium.dev";
+        element.sendKeys(expectedName + Keys.ENTER);
+        driver.findElement(By.xpath("//a[@href='https://www.selenium.dev/']")).click();
```
Comment (David91) [inline seleniumTest.java:25]: Why you enter google.com and then after several steps selenium.dev ?
Enter selenium.dev directly.

File: seleniumTest.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        driver.findElement(By.xpath("//a[@href='/downloads']")).click();
+        Thread.sleep(3000);
+        WebElement versionElement = driver.findElement(By.xpath("//a[text()='3.141.59']"));
```
Comment (David91) [inline seleniumTest.java:28]: Although this selector works now, but it's not correct, as You didn't find latest stable version, but a link, which contains text '3.141.59',which can be in another place of page, So find the exact link, which shows latest stable version, not a link, which text matches with the latest stable version link text.

File: seleniumTest.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        String actualVersion = versionElement.getText();
+        String expectedVersion = "3.141.59";
+        Assert.assertEquals(actualVersion, expectedVersion);
```
Comment (David91) [inline seleniumTest.java:31]: Add meaningful error message.

File: seleniumTest.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        WebElement elem = driver.findElement(By.xpath("//input[@name='search']"));
+        String expectedSearch = "selenium webdriver";
+        elem.sendKeys(expectedSearch + Keys.ENTER);
+        driver.findElement(By.partialLinkText("Selenium WebDriver"));
```
Comment (David91) [inline seleniumTest.java:35]: This also works, but strongly depends on writing style of selenium webdriver. So if text will be "Selenium Webdriver", your test will fail. So try another approach for finding a link with your searched "selenium webdriver" text.

File: seleniumTest.java [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        Thread.sleep(3000);
+        //  driver.quit();
```
Comment (David91) [inline seleniumTest.java:37]: Remove the latest Thread sleep and uncomment driver.quit

**Review by David91 — CHANGES_REQUESTED** [review]

File: seleniumTest.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class seleniumTest {
```
Comment (chandranik90) [inline seleniumTest.java:9]: This is a test class, and it should be under test folder

**Review by chandranik90 — CHANGES_REQUESTED** [review]
chrome dirver file should be kept in resources folder and not in project root

**Pull request closed without merging**